cormorack/notebooks
# top-down and 3D plot schema

## Overview

 * Test for producing figures of rose plots and 3D plots. Includes animation for 3D plot.

### Demo

![plot](https://user-images.githubusercontent.com/38630604/39414791-988f9e4a-4bf0-11e8-9a9d-d5238c2ba7be.PNG)

### Notes

 * Uses test data, no real-world numbers here.
 * ignore checkpoint files

## Testing Instructions

 * Run cell in 3D test notebook. Output in Jupyter editor prints the first plot and then a series of memory addresses. Will this also occur in data portal? If so, how can we update figure image to animate a 3D figure? (See demo image)
